README.md

```diff
@@ -1,16 +1,611 @@
+| `iss`      | Delegator: transfer authority (active) | Invoker: request action (active)  |
+| `aud`      | Delegate: gain authority (passive)     | Executor: perform action (active) |
+
+## 2.1 Invoker
+
+The invoker signals to the executor that a task associated with a UCAN SHOULD be performed.
+
+The invoker MUST be the UCAN delegator. Their DID MUST be authenticated in the `iss` field of the contained UCAN.
+
+## 2.2 Executor
+
+The executor is directed to perform some task described in the UCAN by the invoker.
+
+The executor MUST be the UCAN delegate. Their DID MUST be set the in `aud` field of the contained UCAN.
+
+# 3 Envelope
+
+The invocation envelope is a thin wrapper around a UCAN that conveys that all of the contained UCAN actions SHOULD be performed.
+
+All of the roles from the referenced UCANs MUST persist to the invocation per the [roles table](#2-roles).
+
+The invocation wrapper MUST be signed by the same principal that issued the UCAN.
+
+## 3.1 Fields
+
+| Field         | Type                                         | Value     | Description                                      | Required | Default |
+|---------------|----------------------------------------------|-----------|--------------------------------------------------|----------|---------|
+| `ucan/invoke` | `CID`                                        |           | The CID of the UCAN to invoke                    | Yes      |         |
+| `v`           | `SemVer`                                     | `"0.1.0"` | SemVer of the UCAN invocation this schema        | Yes      |         |
+| `run`         | `"*" or {String: {URI : {Ability : [Any]}}}` |           | Which UCAN capabilities to run                   | No       | `"*"`   |
+| `nnc`         | `String`                                     |           | A unique nonce, to distinguish each invocation   | Yes      |         |
+| `ext`         | `Any`                                        |           | Non-normative extended fields                    | No       | `null`  |
+| `sig`         | `Bytes`                                      |           | Signature of the rest of the field canonicalized | Yes      |         |
+  
+### 3.1.1 Invocation
+
+The `ucan/invoke` field MUST contain CIDs pointing to the UCANs to invoke. The outmost UCAN being invoked SHOULD NOT contain any actions that are not intended to be executed.
+
+### 3.1.2 Version
+
+The `v` field MUST contain the version of the invocation object  schema.
+
+### 3.1.3 Run Capabilities
+
+The OPTIONAL `run` field MUST reference the actions contained in the UCAN are to be run during the invocation. To run all actions in the underlying UCAN, the `"*"` value MUST be used. If only specific actions (or [pipelines](#5-promise-pipelining)) are intended to be run, then they MUST be prefixed with an arbitrary label and treated as a UCAN attenuation: all actions MUST be backed by a matching capability of equal or lesser scope.
```

Comment:
> all actions MUST be backed by a matching capability of equal or lesser scope.

Should this be "of equal or greater scope"?

Reply:
Indeed, I have it backwards; thanks!